packages/cli-tools/test/storage-node.test.ts

```diff
@@ -30,4 +31,24 @@ describe('storage node', () => {
         const outputLines = await runCommand('storage-node list')
         expect(outputLines.join()).toMatch(DOCKER_DEV_STORAGE_NODE.toLowerCase())
     })
+
+    it('register storage node, show info, and finally unregister', async () => {
+        const privateKey = await createTestPrivateKey({ gas: true })
+        const publicAddress = new Wallet(privateKey).address
+
+        const urls = 'http://foobar.com,http://foobar.org'
+        await runCommand(`storage-node register ${urls}`, {
+            privateKey
+        })
+
+        const outputLines = await runCommand(`storage-node show ${publicAddress}`)
+        expect(outputLines.join()).toContain('http://foobar.com')
+        expect(outputLines.join()).toContain('http://foobar.org')
+
+        await runCommand('storage-node unregister', {
+            privateKey
+        })
+        const outputLines2 = await runCommand('storage-node list')
+        expect(outputLines2.join()).not.toContain(publicAddress.toLowerCase())
```

Comment:
Maybe the test could assert these after line 43:
```ts
        const outputLines2 = await runCommand('storage-node list')
        expect(outputLines2.join()).not.toContain(publicAddress.toLowerCase())
```

And here assert what the `storage-node show ${publicAddress}` returns: do we expect empty output, or e.g. an error?

Reply:
Addressed in 394bb102d8243457c6bc01c8a1f76e93d7ccd737